.github/workflows/example_test.yaml

```diff
@@ -52,7 +52,7 @@ jobs:
     strategy:
       fail-fast: true
       matrix:
-        no: [1, 2, 3, 4, 5]
+        no: [1, 2, 3, 4, 5, 6, 7, 8, 9, 10]
```

Comment:
**Suggestion:** If the number of iterations is not dependent on any dynamic factor, consider using a single job with a loop inside it instead of creating multiple jobs. This can reduce the overhead of job creation and management.
```suggestion
        no: [1]
```